src/Propel/Runtime/Adapter/SqlAdapterInterface.php

```diff
@@ -55,10 +55,11 @@ public function cleanupSQL(&$sql, array &$params, Criteria $values, DatabaseMap
      * @param string $sql
      * @param int $offset
      * @param int $limit
+     * @param \Propel\Runtime\ActiveQuery\Criteria|null $criteria
      *
      * @return void
      */
-    public function applyLimit(&$sql, $offset, $limit);
+    public function applyLimit(&$sql, $offset, $limit, $criteria = null);
```

Comment:
Oh, context for the change is that this method was declared here as
```php
public function applyLimit(&$sql, $offset, $limit);
```
and was implemented like this in most adapters, but not in `OracleAdapter`, where it is implemented like this:
```php
public function applyLimit(&$sql, $offset, $limit, $criteria = null)
```
Which confused me, since I had not looked at `OracleAdapter`, but saw the method being called with a fourth parameter, which should be wrong, according to the interface.
So I just added the parameter to reflect the actual signature of the method. But it is implemented as given here, without a type hint. I don't mind adding it, but it does mean adding it to existing public methods, and you guys usually don't want to do that. Should I still add it?

Reply:
I think this is fine then as is.